source/changelog.txt

```diff
@@ -12,6 +12,23 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.2.5
+------
+
+*Released April 22, 2024*
+
+Performance improvements:
+
+- :issue:`MONGOSH-1759` – Updates the policy regarding the balance between 
```

Comment:
Could we reword this to:
Improves mongosh startup time time by ~40% by disabling startup snapshot compression in boxednode.

Reply:
Updated.

Reply:
Sorry @GaurabAryal, I just wanted to confirm we wanted the second sentence updated?